src/js/va-letters/containers/VALettersApp.jsx

```diff
@@ -1,11 +1,30 @@
 import React from 'react';
 import { connect } from 'react-redux';
 
+import FormTitle from '../../common/schemaform/FormTitle';
+import { getCurrentFormStep } from '../../common/utils/helpers';
+import SegmentedProgressBar from '../../common/components/SegmentedProgressBar';
+import NavHeader from '../../common/components/NavHeader';
+
+import { chapters } from '../routes';
+
 class VALettersApp extends React.Component {
   render() {
-    const { children } = this.props;
+    const { children, location } = this.props;
+
     return (
-      <div>{children}</div>
+      <div className="usa-grid">
+        <div className="usa-width-two-thirds medium-8 columns">
```

Comment:
`medium-8` is a Foundation thing (I think), should we make an effort to use just the uswds grid in new code if possible?